src/_data/whats-new.yml

```diff
@@ -4,8 +4,16 @@ description: This page contains recent changes that we think you'd like to know
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: "/whats-new.html"
 thread: "/whatsnew-feed.xml"
-updated: Wed Dec 16 16:19:47 2020
+updated: Mon Jan 4 22:27:24 2021
 entries:
+- description: Added a new topic with instructions for[ reverting from a split database
```

Comment:
@jeff-matthews please fix description in your PR

Reply:
done